package.json

```diff
@@ -25,28 +28,31 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "files": [
-    "*.js",
-    "mixins",
-    "vendor-*",
+    "dist",
+    "index.js",
+    "index.es-modules.js",
     "bin"
   ],
   "engines": {
-    "node": ">=3.3"
+    "node": ">=4.7"
   },
   "dependencies": {
     "axios": "^0.15.3",
-    "babel-loader": "^6.2.10",
+    "es6-promise": "^4.0.5",
     "lodash": "^4.17.4",
-    "qs": "^6.1.0",
-    "webpack": "^1.14.0"
+    "qs": "^6.1.0"
   },
```

Comment:
can you add 
```
"publishConfig": {
  "tag": "alpha"
}
```
so when we merge to master the the auto release will still release under `alpha` or maybe we can promote to `beta`